src/implementations/vanilla/components/list-illustration/_list-illustration.scss

```diff
@@ -185,6 +185,14 @@ $_icon_color: null !default;
   }
 }
 
+@include breakpoints.up('m') {
```

Comment:
but why do we have this css?

```
.ecl-list-illustration--zebra {
    -webkit-margin-start: -50vw;
    left: 50%;
    margin-inline-start: -50vw;
    position: relative;
    width: 100vw;
}
```
what is this for? The proposed change fixes the main issue with our demos, but still it feels we shouldn't force the element to take full width even in mobile and tablets, potentially we could be doing this still in a layout with columns, for instance, and in any case it's not really clear what is that we want to achieve with that.

Reply:
this was a request from COMM, to have the zebra list look less like cards when on mobile...
![image](https://github.com/ec-europa/europa-component-library/assets/25579221/5d674993-c124-4337-9d96-fe41eccdaf03)

Reply:
but if you think that it would be too dangerous to keep it, we can put it away, at least for now

Reply:
did not really get the "comparison" with cards, but anyway, like this is surely safer, if we limit it to the mobile breakpoint.
Just to understand, this is done to ensure that the grey background of the zebra would cover the full width of the viewport regardless of a container that could set a padding, basically.

Reply:
yes basically this is to have no white padding around the grey area, so it looks more like a vertical list of items